src/db/cassandra/file_iterator.py

```diff
@@ -0,0 +1,36 @@
+class FileIterator:
```

Comment:
This class seems to serve the purpose of file.read(size) but with the disadvantage of handling everything in memory. This is a problem because video files are usually very large. https://docs.python.org/3/tutorial/inputoutput.html#methods-of-file-objects.

I would recommend either removing or rewriting this class to use file.read